<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataDown</class>
 <widget class="QWidget" name="DataDown">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>771</width>
     <height>761</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>371</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>服务器目录</string>
    </property>
    <widget class="QLineEdit" name="RemoteFileDir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="RemoteListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>351</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>连接服务器</string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>721</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="RRMRadio">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>RRM</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BCNRadio">
     <property name="geometry">
      <rect>
       <x>197</x>
       <y>25</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>BCN</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="PPNRadio">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>25</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>PPN</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="TRKRadio">
     <property name="geometry">
      <rect>
       <x>378</x>
       <y>25</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>TRK</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>54</y>
       <width>72</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>IP地址：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>327</x>
       <y>54</y>
       <width>72</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IPLlineEdit">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>55</y>
       <width>188</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="NameLineEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>55</y>
       <width>188</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="connectBtn">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>54</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>90</y>
       <width>72</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>端口号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PWLineEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>90</y>
       <width>188</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>90</y>
       <width>54</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnetBtn">
     <property name="geometry">
      <rect>
       <x>628</x>
       <y>90</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>disconnect</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PortLineEdit">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>90</y>
       <width>188</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>361</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>本地目录</string>
    </property>
    <widget class="QLineEdit" name="LocalFileDir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="LocalListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>341</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="downBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>710</y>
      <width>99</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>DownLoad</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>710</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
